net/third_party/quiche/src/quiche/quic/core/quic_constants.h

```diff
@@ -294,8 +294,12 @@ inline constexpr int kDefaultIetfLossDelayShift = 3;
 
 // Maximum number of retransmittable packets received before sending an ack.
 inline constexpr QuicPacketCount kDefaultRetransmittablePacketsBeforeAck = 2;
-// Wait for up to 10 retransmittable packets before sending an ack.
+// Wait for up to 50 retransmittable packets before sending an ack.
+#if defined(USE_COBALT_CUSTOMIZATIONS)
+inline constexpr QuicPacketCount kMaxRetransmittablePacketsBeforeAck = 50;
```

Comment:
Now that you've introduced `set_max_retransmittable_packets_before_ack`, should we call that from cobalt instead of changing the default value? We then wouldn't need change the tests at all

Actually, thinking about it, do we need to override it in Cobalt explicitly to be 50? Can't we just use one of the added quic tags instead and take advantage of the plumbing you already added?

Reply:
We talked, makes sense to change the default—we could still potentially use `set_max_retransmittable_packets_before_ack` to set the default via Cobalt, which may be nice as we wouldn't need to change the unit tests in that case